de.dashup.shared/src/main/java/de/dashup/shared/LayoutModeStructure.java

```diff
@@ -0,0 +1,29 @@
+package de.dashup.shared;
+
+import java.util.List;
+
+public class LayoutModeStructure {
```

Comment:
Is this class necessary? The user has a list of sections, that again have a list of panels. Why not using this? Is there a reason for this?

Reply:
The class contains two lists of sections, one for the new layout and one which are to be deleted from database. Furthermore spring will convert the json send by the frontend via a post request to an object of this class. Without the class it would lead to problems.